praw/const.py

```diff
@@ -56,6 +56,7 @@
     'list_wikicontributor':   'r/{subreddit}/about/wikicontributors/',
     'live_contributors':      'live/{id}/contributors',
     'live_invite':            'live/{id}/invite_contributor',
+    'live_leave':             'api/live/{id}/leave_contributor',
```

Comment:
Weird that this has `api` in the path.

Actually, according the the docs, all the other paths should have `api` in them as well. Is that optional?